tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -113,21 +112,28 @@ public void RunAllTests()
         public void AcquireTokenTest()
         {
             TestCommon.ResetInternalStaticCaches();
-            _mobileTestHelper.AcquireTokenInteractiveTestHelper(
+
+            _mobileTestHelper.AcquireTokenTestHelper(
                 _xamarinController,
-                LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult());
+                LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult(),
+                CoreUiTestConstants.AcquireTokenInteractive);
         }
 
         /// <summary>
         /// Runs through the standard acquire token flow
         /// </summary>
         [Test]
+        [Ignore("needs some work")]
```

Comment:
Do we need this test? There is a consent test in desktop tests.

Reply:
i agree. removed.

---
In reply to: [300986015](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1255#discussion_r300986015) [](ancestors = 300986015)